tests/test_spherePoint.py

```diff
@@ -762,28 +771,12 @@ def testRotatedAlias(self):
             self.assertAlmostEqual(0.0, newPoint.getLongitude().asDegrees())
             self.assertAlmostEqual(80.0, newPoint.getLatitude().asDegrees())
 
-    def testOffsetError(self):
-        """Test if offset() correctly handles invalid input.
-        """
-        northPole = SpherePoint(0.0*degrees, 90.0*degrees)
-        southPole = SpherePoint(0.0*degrees, -90.0*degrees)
-        nonSingularity = SpherePoint(0.0*degrees, 45.0*degrees)
-
-        with self.assertRaises(pexEx.DomainError):
-            northPole.offset(-90.0*degrees, 10.0*degrees)
-        with self.assertRaises(pexEx.DomainError):
-            southPole.offset(90.0*degrees, 0.1*degrees)
-        with self.assertRaises(pexEx.InvalidParameterError):
-            nonSingularity.offset(90.0*degrees, -0.1*degrees)
-
     def testOffsetValue(self):
         """Test if offset() returns the expected value.
         """
         # This should cover arcs over the meridian, across the pole, etc.
         for lon1, lat1 in self._dataset:
             point1 = SpherePoint(lon1, lat1)
-            if point1.atPole():
-                continue
```

Comment:
Again, please add some test coverage at the poles (including testing for consistency between `offset` and `bearingTo`/`separation`).

Reply:
Done in testOffsetValues